ext/release-lead.rake

```diff
@@ -1,17 +1,38 @@
 require 'json'
 require 'optparse'
+require 'yaml'
 
 namespace :release_lead do
   desc "Find platforms added and removed between releases"
   task :platform_diff, [:from, :to] do |t, args|
     abort ('`from` argument is required') unless args.from
     abort ('`to` argument is required') unless args.to
+    puts "Platform Changes:"
+    puts
     puts `git diff --summary #{args.from}..#{args.to} configs/platforms`.
         gsub('configs/platforms/', '').gsub(/mode \d+ /, '').
         gsub('delete', 'Retired').gsub('create', 'Added').
         gsub('rename', 'Renamed')
   end
 
+  desc "List currently supported platforms on the current branch"
+  task :supported_platforms do |t, args|
+    puts "Currently Supported Platforms:"
+    puts
+    Dir.glob("./configs/platforms/*").map { |platform_file| File.basename(platform_file).gsub(".rb",'') }.each do |_platform|
```

Comment:
I'd add a few line-breaks to split this logic up. Namely using `do` rather than `{}` in the `map` block.